name/templates/name/_pagination.html

```diff
@@ -2,23 +2,23 @@
     <div class="col-sm-12">
         <div class="text-center">
             <ul class="pagination">
-                {% if entries.number != 1 %}
+                {% if page_obj.number != 1 %}
                     <li><a href="?q_type={{ q_type }}&amp;q={{ q }}&amp;order={{ sort }}&amp;page=1">first</a></li>
```

Comment:
All taken care of now. I reduced the number if results per page to 1, to enable pagination, then tested this out.